ocs_ci/ocs/platform_nodes.py

```diff
@@ -818,6 +824,271 @@ def _prepare_node(self, node_id):
             except Exception:
                 logger.error("Failed to create RHEL node")
                 raise
+        elif self.node_type == 'RHCOS':
+            conf = self._prepare_rhcos_node_conf()
+            conf['node_id'] = node_id
+            try:
+                self.aws_instance_obj = self._prepare_upi_rhcos_node(conf)
+            except Exception:
```

Comment:
Is there a more specific `Exception` we can except here?

Reply:
this function can expect wide range of exceptions from stack creation, s3_bucket operations hence made it as generic.